sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java

```diff
@@ -126,13 +158,23 @@ public FavoritesService favorites() {
   }
 
   @Override
-  public QualityProfilesService qualityProfiles() {
-    return qualityProfilesService;
+  public QualityProfilesService qualityProfilesOld() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "QualityProfilesService"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)